### PR TITLE
Authorization Server Obligations

### PR DESCRIPTION
**Pull request by ghost**
Changed section title and incorporated two sub-sections:  Signature Verification and Issuing Access Tokens.  Note that the table in the Issuing Access Tokens section was moved down from the original "Obtaining an Access Token" section.  Pull Request #90 (Obtaining an Access Token) deleted this table, so to avoid confusion, I did not delete it again here.

### CODE REVIEW

File: authorization.md [hunk 1]
```diff
@@ -259,7 +260,54 @@ matches the value supplied at registration time for the specified `client_id`).
...
+    </tr>
+  </tbody>
+</table>
+
+Access tokens issued under this profile SHALL be short-lived; the `expires_in` 
+value SHOULD NOT exceed `300`, which represents an expiration-time of five minutes.  
+
+If an error is encountered during the authorization process, the server SHALL
+respond with the appropriate error message defined in [Section 5.2 of the OAuth 2.0 specification](https://tools.ietf.org/html/rfc6749#page-45).  The server SHOULD include an 
+`error_uri` and `error_description` as defined in OAuth 2.0.
```
Comment (michelemottini) [inline authorization.md]: Why both uri and description? ...and aren't they optional?

File: authorization.md [hunk 2]
```diff
@@ -259,7 +260,54 @@ matches the value supplied at registration time for the specified `client_id`).
...
+      <td><span class="label label-success">required</span></td>
+      <td>The lifetime in seconds of the access token. The recommended value is <code>300</code>, for a five-minute token lifetime.</td>
+    </tr>
+    <tr>
+      <td><code>scope</code></td>
+      <td><span class="label label-success">required</span></td>
+      <td>Scope of access authorized. Note that this can be different from the scopes requested by the app.</td>
+    </tr>
+  </tbody>
+</table>
```
Comment (michelemottini) [inline authorization.md:303]: You can do tables using markdown: https://help.github.com/articles/organizing-information-with-tables/

File: authorization.md (hunk 1 above)
Comment (jmandel) [reply authorization.md]: Agreed, I'd like to modify like:

```suggestion
`error_uri` or `error_description` as defined in OAuth 2.0.  
```
@bakerdb if this make sense, I think you can apply this change by taking this suggestion inline.

File: authorization.md (hunk 2 above)
Comment (jmandel) [reply authorization.md:303]: This is true, but there are formatting limitations (e.g., newlines, code blocks), and we've historically had better luck with HTML here. Could be time to re-visit, but let's address that question out of band.

**Review by jmandel — APPROVED** [review]

File: authorization.md [hunk 3]
```diff
@@ -259,7 +260,56 @@ matches the value supplied at registration time for the specified `client_id`).
...
+the [OAuth 2.0 specification](https://tools.ietf.org/html/rfc6749#section-5.2). 
+
+### Issuing Access Tokens
+
+Once the backend service has been authenticated, the authorization server SHALL
+mediate the request to assure that the scope requested is within the scope pre-authorized
+to the backend service.
+
+If the access token request is valid and authorized, the authorization server
+SHALL issue an access token in response.  The access token SHALL be a JSON object with
```
Comment (jmandel) [inline authorization.md]:
```suggestion
SHALL issue an access token in response.  The access token response SHALL be a JSON object with 
```

**Review by jmandel — CHANGES_REQUESTED** [review]
Just a small wording change requested.

**Review by jmandel — APPROVED** [review]

**Pull request merged**